### PR TITLE
Machine controller deployment

### PR DESCRIPTION
**Pull request by csrwng**
Adds a make target to build the aws-machine-controller image
Modifies the cluster-api deployment playbook to include the aws-machine-controller

### CODE REVIEW

**Comment by csrwng** [comment]
@dgoodwin @joelddiaz ptal

File: build/cluster-operator-ansible/playbooks/cluster-api-prep/files/cluster-api-template.yaml [hunk 1]
```diff
@@ -333,3 +384,10 @@ parameters:
 # location of container image
 - name: IMAGE
   value: quay.io/openshift/kubernetes-cluster-api:latest
+# machine controller image
+- name: MACHINE_CONTROLLER_IMAGE
+  value: quay.io/csrwng/aws-machine-controller:latest
```
Comment (dgoodwin) [inline build/cluster-operator-ansible/playbooks/cluster-api-prep/files/cluster-api-template.yaml]: Looks like this should be openshift here.

Comment (csrwng) [reply build/cluster-operator-ansible/playbooks/cluster-api-prep/files/cluster-api-template.yaml]: @dgoodwin it's what will work for now. It seems I can no longer create image repos under the openshift org in quay.io

Comment (dgoodwin) [reply build/cluster-operator-ansible/playbooks/cluster-api-prep/files/cluster-api-template.yaml]: Gotcha, that is weird.

**Pull request merged**